### PR TITLE
XMAS Livery (Updated)

### PR DESCRIPTION
**Pull request by ko-zone**
Added XMAS Livery

## Summary of Changes

Default XMAS Livery Downloaded with:
1. Added comp texture for fuse and wing (remove roughness from default asobo livery)
2. Added corrected style thumbnails. One by one like default aircraft.

Discord username (if different from GitHub): 
KO Zone#2134
or
Mugz#6174

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO-LIVERY/aircraft.cfg [hunk 1]
```diff
@@ -8,7 +8,7 @@ base_container = "..\Asobo_A320_NEO"
 ;===================== FLTSIM =====================
 
 [FLTSIM.0]
-title = "FlyByWire A320 NEO" ; Variation name
+title = "FlyByWire A320neo" ; Variation name
```
Comment (lukecologne) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO-LIVERY/aircraft.cfg:11]: If you're already at it, you could probably correct the name for the "default" A320 livery as well

**Comment by ChruutvoLuzi** [comment]
Category: QA Tester
Name: ChruutvoLuzi | SD-PB.ch#8902
Date of testing: 2.12.2020
Version of the sim: 1.11.6.0
PR Tested: #2181 
Object of test: XMAS Livery

Testing technique: Description of how you tested the Pull Request
Steps you've made to test the Pull Request:
1.  Selected the A320 in the Hangar
2. Switched to both FBW Liveries

Issues (If Any): Not really a big issue but the Naming and Thumbnails are a bit misleading. Both have the same Thumbnail and Name
Microsoft Flight Simulator Screenshot 2020 12 02 - 17 43 50 71
Overall Rating: Good, except the two mentioned issues
Conclusions of the testing: Works as described besides the issue mentioned above

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO-LIVERY/aircraft.cfg (hunk 1 above)
Comment (ko-zone) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO-LIVERY/aircraft.cfg:11]: pushing the new fix right now!

**Comment by Awemeter** [comment]
Category: Trial QA tester
Name: Awemeter#7919
Date of testing: 12/2/2020
Version of the sim: 1.11.6.0
PR Tested: #2181 
Object of test: Xmas Livery

Testing technique: Went into the sim to check if Xmas livery worked properly.
Steps you've made to test the Pull Request:
1. Loaded into the sim. Went into the hanger and checked to see if livery was showing up with right thumbnail and name.
2.  Spawned airplane on the runway to check if livery showed in sim.



Flight notes (If required):

Issues (If Any): Noticed that with the Xmas pr installed the airbus livery says airbus house but with the current dev just says default.
Overall Rating: Good
Conclusions of the testing: Just want to make sure that the default airbus is supposed to say house instead of the default. other than that it's good to be merged.

**Comment by ko-zone** [comment]
@Awemeter yep its meant to be airbus house great to know all works now

**Comment by lukecologne** [comment]
Alright then, looks good, going to merge

**Pull request merged**